src/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -14,6 +18,8 @@ namespace AzureMcp.Extensions;
 
 public static class OpenTelemetryExtensions
 {
+    private const string DefaultAppInsights = "InstrumentationKey=21e003c0-efee-4d3f-8a98-1868515aa2c9;IngestionEndpoint=https://centralus-2.in.applicationinsights.azure.com/;LiveEndpoint=https://centralus.livediagnostics.monitor.azure.com/;ApplicationId=f14f6a2d-6405-4f88-bd58-056f25fe274f";
```

Comment:
Should this be loaded or should this be in code?

Reply:
Do you think this would ever change?  I'm thinking if embedded we have no way to update it.  But I guess that is also the case unless we pull it dynamically from a server.

Reply:
I don't think is going to change... I picked up this approach from: 

https://github.com/Azure/azure-dev/blob/0f2d6804c3edd262ebd33804c897eaa45ec0cde6/cli/azd/internal/telemetry/telemetry.go#L39